object-browser.md

```diff
@@ -0,0 +1,55 @@
+```hbs
+{{frost-object-browser
+  items=<sorted-filtered-selected-expanded-items>
+  header=(component ‘frost-info-bar’
+    icon=(hash
+      name=‘foo’
+      pack=‘foo’
+    )
+    title=‘foo’
+    description=‘foo’ or (component ‘foo’ …
+    scope=(component ‘foo’ ...
```

Comment:
Scope is the term we're using to refer to the mechanism (keyword, time range, etc.) used to focus the overall context of the data prior to really interacting with the object browser.  This is used to limit the number of facets initially presented to a user, since facets are based on the data set.